apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -685,7 +740,7 @@ static void tc_fs_vfs_readdir(void)
 */
 static void tc_fs_vfs_rewinddir(void)
 {
-	int ret, count;
+	int ret, count, temp_count;
```

Comment:
Coding rule does not encourage multiple variable declaration of a type in a single line.

Reply:
Okay , will change it